test/studies/labelprop/labelprop-tweets.chpl

```diff
@@ -364,7 +364,8 @@ proc create_and_analyze_graph(Pairs)
   var G = buildUndirectedGraph(triples, false, {1..max_nid} );
 
   // Clear out memory used by triples.
-  triples.domain.clear();
+  // LK: this should not be allowed
+  // triples.domain.clear();
```

Comment:
This code has occasionally been shared out to people interested in this style of computation, so I'd suggest removing both lines rather than leaving them in as comments.